descriptions/x-terminal-emulator.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=X Terminal Emulator
+Comment=Default Terminal Emulator to use when running text-based applications, mainly from the Debian menu.
+Type=Application
+Icon=x-terminal-emulator
```

Comment:
Renaming as .desktop files might cause confusions since they are not really desktop files. What do you think?